spec/scrabble_player_class_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "./lib/scrabble_player_class"
+
+describe Player do
+  before :each do
+    @player = Scrabble::Scrabble::Player.new("Jack")
+  end
+
+
+  describe "returns the @name instance variable" do
+    it "returns @name" do
+      expect(@player.name).to eq "Jack"
+    end
+  end
+
+  describe "shows words played" do
+    it "returns @plays" do
+      expect(@player.play("apple")).to eq ["apple"]
+    end
+  end
+
+
+  describe "shows total score" do
+    it "returns score of all words played" do
+      @player.play("venus")
+      @player.play("zzzzzzzzzzzzzz")
+      @player.play("a")
+      expect(@player.total_score).to eq 7
+    end
+  end
+
+  describe "tells if the player has over 100 points" do
+    it "returns ture or false" do
+      @player.play("zzzzzzzzzzzzzz")
+      expect(@player.won?).to be false
+    end
+  end
+
+  describe " tells the highest scoring word" do
+    it "returns the highest scoring word" do
+      @player.play("venus")
+      @player.play("zzzzzzzzzzzzzz")
+      @player.play("a")
+
+      expect(@player.highest_scoring_word).to eq "a"
```

Comment:
I'm confused about how this is the highest scoring word. Am I missing something?